README.md

```diff
@@ -1,19 +1,32 @@
 
-# 1. Install prerequisites on Ubuntu 20.04
+# 1. Install prerequisites on Ubuntu 20.04 and 22.04
 
 ```
-    $ sudo apt install libunwind-dev libgflags-dev libssl-dev libelf-dev protobuf-compiler cmake zstd clang g++
+    $ sudo apt install libunwind-dev libgflags-dev libssl-dev libelf-dev protobuf-compiler cmake libzstd-dev clang g++
 ```
 
-## 2 Build autofdo tools using clang-10
+## 1.1 For Ubuntu 20.04 users, the cmake version (3.16.3) is too old to build third_party/llvm, you can upgrade your cmake by following steps in https://askubuntu.com/questions/355565/how-do-i-install-the-latest-version-of-cmake-from-the-command-line
+
+```
+    $ sudo apt purge --auto-remove cmake
+    $ wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
+    $ sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main'     
+    $ sudo apt update && sudo apt install cmake
+```
+
+## 2 Build autofdo tools
 
 ```
     $ git clone --recursive --depth 1 https://github.com/google/autofdo.git
     $ git checkout v0.20.1
     $ cd autofdo
     $ mkdir build
     $ cd build 
-    $ cmake -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_BUILD_TYPE=Release ../
+    $ # Build LLVM tools for AUtoFDO and Propeller
+    $ cmake -DENABLE_TOOL=LLVM -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_BUILD_TYPE=Release ../
```

Comment:
Can you drop the version suffix, i.e `-10` from clang? That makes the instructions generic for 20.04 and 22.04. 

Also update the gcc text below to drop the suffixes.

Reply:
Done dropping the version suffix for clang/gcc.